src/ucp/core/ucp_mm.h

```diff
@@ -117,6 +117,16 @@ typedef struct {
 } ucp_mem_dummy_handle_t;
 
 
+/**
+ * Memory type pack/unpack registration context
+ */
+typedef struct {
+    ucp_mem_h         ucp_memh;    /* memh from rcache if MD id cacheable */
+    uct_mem_h         uct_memh;    /* memh for specific MD */
+    uct_rkey_bundle_t rkey_bundle; /* rkey bundle from memh */
+} ucp_mtype_pack_context_t;
```

Comment:
would it make sense to add field `md_index` to this struct?
I see it's being used pretty often together